src/App.jsx

```diff
@@ -2,6 +2,10 @@ import { CssBaseline } from "@mui/material";
 import { ThemeProvider } from "@mui/system";
 import { Store } from "./Components/Store";
 import { createTheme } from "@mui/material/styles";
+import { ThemeProvider } from "styled-components";
```

Comment:
Vovê importou o `ThemeProvider` aqui mas não utilizou.

Reply:
Já existe um `ThemeProvider` nesse arquivo, na linha 2:
![image](https://user-images.githubusercontent.com/54550926/160853016-87752c3e-0283-4452-b34c-eca30b84fc3e.png)

Da para trocar o nome desse `ThemeProvider` do styled components:
 ```suggestion
import { ThemeProvider as StyledThemeProvider } from "styled-components";
```